src/routes/AuthorityViewRoute/AuthorityViewRoute.test.js

```diff
@@ -1,6 +1,9 @@
 import { render } from '@folio/jest-config-stripes/testing-library/react';
 
-import { useMarcSource } from '@folio/stripes-authority-components';
+import {
+  useMarcSource,
+  useAuthority,
```

Comment:
Shouldn't all requests be with the central tenant id for the `shared` record upon opening the authority detail view, not just these two?

Reply:
You're right, created a new PR to set tenantId when fetching linking rules when viewing source record
https://github.com/folio-org/ui-quick-marc/pull/580

Reply:
Is it supposed to do the same for:
- mapping-rules/marc-authority
- authority-source-files
- linking-rules/instance-authority

Reply:
![image](https://github.com/folio-org/ui-marc-authorities/assets/77053927/8021048e-8e7f-4bf4-a72d-b9c0395c3624)

Reply:
It looks like not all requests leverage central tenant id.

https://github.com/folio-org/ui-marc-authorities/assets/77053927/8a0060cd-6571-49f3-abad-442de4327002